README.md

```diff
@@ -12,13 +12,23 @@ The [Atomic Developer Bundle](https://github.com/projectatomic/adb-atomic-develo
 
 ## Quick Start
 
-1. Install `vagrant-service-manager` plugin
+1. Install `vagrant-service-manager` plugin:
 
         vagrant plugin install vagrant-service-manager
 
-2. Get the [Vagrantfile](Vagrantfile) and start the ADB using `vagrant up`. [More](https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/docs/installing.rst) documentation on setting up ADB.
+2. Get the [Vagrantfile](Vagrantfile).
 
-3. Run the plugin to get environment variables and certificates
+3. Enable your desired service(s) in [Vagrantfile](Vagrantfile) as:
+
+        config.servicemanager.services = 'openshift'
+
+   *Note*
+
+   `docker` is default service and does not require above configuration.
+
+4. Start the ADB using `vagrant up`. [More](https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/docs/installing.rst) documentation on setting up ADB.
```

Comment:
Suggestion:

4. Start the ADB using `vagrant up`. Users of the ADB may wish to consult the [Installation Documentation](https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/docs/installing.rst).